test/Escrow.js

```diff
@@ -99,6 +99,18 @@ contract('Escrow', (accounts) => {
         assert(false);
       }
     });
+
+    it('transfer tokens to canceler if contract funded when abort is called', async () => {
+      try {
+        await HMT.transfer(Escrow.address, 100, { from: canceler });
+        assert.equal((await HMT.balanceOf.call(Escrow.address)).toNumber(), 100, "Escrow has not been properly funded.");
+        tx = await Escrow.abort({ from: canceler });
+        console.log(`Abort costs: ${tx.receipt.gasUsed} wei.`);
+        assert.equal((await HMT.balanceOf.call(Escrow.address)).toNumber(), 0, "Escrow has not been properly aborted.");
```

Comment:
Think you also wanna check the hmt balance of the caller as well here to ensure it gets refunded?

Reply:
Oh - seems like you check that below?